### PR TITLE
Refactor style black

### PR DESCRIPTION
**Pull request by mdiazmel**
Refactoring the code to use `black`. It also uses pre-commit hook to install it automatically.

### CODE REVIEW

File: clinicadl/clinicadl/tools/deep_learning/models/image_level.py [hunk 1]
```diff
@@ -23,41 +25,32 @@ def __init__(self, dropout=0.5, n_classes=2):
             nn.BatchNorm3d(8),
             nn.ReLU(),
             PadMaxPool3d(2, 2),
-
```
Comment (14thibea) [inline clinicadl/clinicadl/tools/deep_learning/models/image_level.py]: I like here to let spaces in order to separate the convolutional blocks...
Is it possible to let them?

File: clinicadl/clinicadl/tools/deep_learning/models/image_level.py [hunk 2]
```diff
@@ -83,45 +77,30 @@ def __init__(self, dropout=0.5, n_classes=2):
...
-        )
+        self.fc_var = nn.Sequential(Flatten(), nn.Linear(128 * 6 * 7 * 6, 1300))
 
         self.classifier = nn.Sequential(
-            nn.Linear(1300, 50),
-            nn.ReLU(),
-
-            nn.Linear(50, n_classes)
-
+            nn.Linear(1300, 50), nn.ReLU(), nn.Linear(50, n_classes)
```
Comment (14thibea) [inline clinicadl/clinicadl/tools/deep_learning/models/image_level.py]: same here could be nice to separate the structures

File: clinicadl/clinicadl/tools/deep_learning/models/patch_level.py [hunk 3]
```diff
@@ -26,36 +27,29 @@ def __init__(self, dropout=0, n_classes=2):
             nn.BatchNorm3d(15),
             nn.ReLU(),
             PadMaxPool3d(2, 2),
-
```
Comment (14thibea) [inline clinicadl/clinicadl/tools/deep_learning/models/patch_level.py]: same comment on the models of this file

File: clinicadl/clinicadl/tools/deep_learning/models/slice_level.py [hunk 4]
```diff
@@ -117,28 +120,24 @@ def __init__(self, dropout=0.5, n_classes=2):
             nn.LeakyReLU(),
             nn.MaxPool2d(2, 2),
             # feature_map : (8@64x64)
-
```
Comment (14thibea) [inline clinicadl/clinicadl/tools/deep_learning/models/slice_level.py]: same comment for the models of this file

File: clinicadl/tests/test_cli.py [hunk 5]
```diff
@@ -4,217 +4,233 @@
...
+            "random",
+            "/dir/caps",
+            "t1-linear",
+            "/dir/output/",
+            "--n_subjects",
+            "10",
+            "--mean",
+            "0.5",
+            "--sigma",
+            "0.5",
```
Comment (14thibea) [inline clinicadl/tests/test_cli.py]: maybe it could also be nice to keep the option value next to the flag?

**Review by 14thibea — APPROVED** [review]
I would only allow some files not to be reformatted by black (cf coments)

File: clinicadl/tests/test_cli.py (hunk 5 above)
Comment (mdiazmel) [reply clinicadl/tests/test_cli.py]: Solved in 60a909a.

File: clinicadl/clinicadl/tools/deep_learning/models/image_level.py (hunk 1 above)
Comment (mdiazmel) [reply clinicadl/clinicadl/tools/deep_learning/models/image_level.py]: Solved in fa9aea6.

**Comment by mdiazmel** [comment]
Hi @14thibea ,

Thanks for you feedback. I added `# fmt: off` and `# fmt: on` labels during the model creation and also in when testing the cli, as you proposed in the review. Shall we merge ?

**Comment by 14thibea** [comment]
Sounds good to me!
